README.md

```diff
@@ -59,4 +58,4 @@ Copyright (c) 2013-2017 Johan Andersson
 
 MIT (See docs/License.txt file)
 
-The license should be the same as redis-py (https://github.com/andymccurdy/redis-py)
\ No newline at end of file
+The license should be the same as redis-py (https://github.com/andymccurdy/redis-py)
```

Comment:
I do not understand what github says that was changed on this line. Do you know? It do not show any content change on the line and no change to end of file O.o really wierd.